### PR TITLE
[ja] Update certificates.md

### PR DESCRIPTION
**Pull request by TheDedster**
Added link at "サーバー認証書" in the japanese version of the document like the english version.

### CODE REVIEW

**Comment by TheDedster** [comment]
make verify

File: content/ja/docs/setup/best-practices/certificates.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Kubernetesでは、TLS認証のためにPKI証明書が必要です。
 Kubernetesは下記の用途でPKIを必要とします：
 
 * kubeletがAPIサーバーの認証をするためのクライアント証明書
+* API サーバーが kubelet と通信するための Kubelet [サーバー認証書](https://kubernetes.io/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#client-and-serving-certificates)
```
Comment (t-inu) [inline content/ja/docs/setup/best-practices/certificates.md:23]:
```suggestion
* APIサーバーがkubeletと通信するためのkubelet[サーバー証明書](https://kubernetes.io/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/#client-and-serving-certificates)
```
I think "証明書" is better.
And, the rules for Japanese pages do not put a space between alphanumeric characters and Japanese words.

**Review by t-inu — COMMENTED** [review]
Thank you for the contribution, and sorry for the late review.

**Comment by t-inu** [comment]
Hi @TheDedster, are you working on this PR?
If there is no response, I will close this.

**Pull request closed without merging**